container/flv/encoder.go

```diff
@@ -166,93 +169,147 @@ func (s *frameScanner) readByte() (b byte, ok bool) {
 	return b, true
 }
 
-// write implements io.Writer. It takes raw h264 and encodes into flv, then
+// WriteVideo takes raw h264 and encodes into flv, then
 // writes to the encoders io.Writer destination.
-func (e *Encoder) Write(frame []byte) (int, error) {
+func (e *Encoder) WriteVideo(videoFrame []byte) (int, error) {
```

Comment:
We've changed this signature but I don't see any changes to the rest of the av code to ensure we haven't broken anything where the Encoder is used ?

Reply:
But why do we need to change this signature ?